app/DoctrineMigrations/Version20181023151546.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace Surfnet\StepupMiddleware\Migrations;
+
+use Doctrine\DBAL\Migrations\AbstractMigration;
+use Doctrine\DBAL\Schema\Schema;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+class Version20181023151546 extends AbstractMigration
+{
+    /**
+     * @param Schema $schema
+     */
+    public function up(Schema $schema)
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->abortIf($this->connection->getDatabasePlatform()->getName() !== 'mysql', 'Migration can only be executed safely on \'mysql\'.');
+
+        $this->addSql('ALTER TABLE ra_listing DROP PRIMARY KEY');
+        $this->addSql('ALTER TABLE ra_listing ADD id INT AUTO_INCREMENT PRIMARY KEY NOT NULL, ADD ra_institution VARCHAR(255) NOT NULL COMMENT \'(DC2Type:institution)\'');
```

Comment:
Nitpick: maybe position the `id` column at the start of the schema?!

Reply:
Fixed